expression/builtin_string_test.go

```diff
@@ -769,6 +769,7 @@ func (s *testEvaluatorSuite) TestSpace(c *C) {
 }
 
 func (s *testEvaluatorSuite) TestLocate(c *C) {
+	// 1. Test LOCATE(substr, str)
```

Comment:
can we merge all the test cases to two tbl, 
locate() non-binary string for one,
and locate() binary string for one.